modules/upi-provisioning-new-hosts-in-a-upi-cluster.adoc

```diff
@@ -22,43 +22,157 @@ To provision bare-metal hosts to the cluster by using the BMO, you must set the
 
 . Create the `Secret` CR and the `BareMetalHost` CR.
 
-.. Save the following YAML in the `bmh.yaml` file:
+. Create a configuration file for the bare-metal node. Depending on whether you are using a static configuration or a DHCP server, use one of the following example `bmh.yaml` files, replacing values in the YAML to match your environment:
++
+[source,terminal]
+----
+$ vim bmh.yaml
+----
+* *Static configuration* `bmh.yaml`:
 +
 [source,yaml]
 ----
 ---
 apiVersion: v1
 kind: Secret
 metadata:
-  name: worker1-bmc
+name: openshift-worker-<num>-network-config-secret <1>
+namespace: openshift-machine-api
+type: Opaque
+stringData:
+ nmstate: | <2>
+  interfaces: <3>
+  - name: <nic1_name> <4>
+    type: ethernet
+    state: up
+    ipv4:
+      address:
+      - ip: <ip_address> <4>
+        prefix-length: 24
+      enabled: true
+  dns-resolver:
+    config:
+      server:
+      - <dns_ip_address> <4>
+  routes:
+    config:
+    - destination: 0.0.0.0/0
+      next-hop-address: <next_hop_ip_address> <4>
+      next-hop-interface: <next_hop_nic1_name> <4>
+---
+apiVersion: v1
+kind: Secret
+metadata:
+name: openshift-worker-<num>-bmc-secret <1>
+namespace: openshift-machine-api
+type: Opaque
+data:
+ username: <base64_of_uid> <5>
+ password: <base64_of_pwd> <5>
+---
+apiVersion: metal3.io/v1alpha1
+kind: BareMetalHost
+metadata:
+customDeploy:
+method: install_coreos
+online: true
+userData:
+name: <openshift-worker-<num> <1>
+namespace: openshift-machine-api
+spec:
+bootMACAddress: <nic1_mac_address> <6>
+bmc:
+address: <protocol>://<bmc_url> <7>
+credentialsName: openshift-worker-<num>-bmc-secret <1>
+disableCertificateVerification: true <8>
+rootDeviceHints:
+deviceName: <root_device_hint> <9>
+preprovisioningNetworkDataName: openshift-worker-<num>-network-config-secret
```

Comment:
```suggestion
metadata:
  name: <openshift-worker-<num>
  namespace: openshift-machine-api
spec:
  online: True
  bootMACAddress: <nic1_mac_address> 
  bmc:
    address: <protocol>://<bmc_url> 
    credentialsName: openshift-worker-<num>-bmc-secret
    disableCertificateVerification: True
  customDeploy:
    method: install_coreos
  online: true
  userData:
    name: worker-user-data-managed 
    namespace: openshift-machine-api
  rootDeviceHints:
    deviceName: <root_device_hint>
```

Reply:
Note on line 12
>To provision bare-metal hosts to the cluster by using the BMO, you must set the `spec.externallyProvisioned` specification in the `BareMetalHost` custom resource to `false`.

`must set` is confusing as we have not set it in the above bmh.yaml. `spec.externallyProvisioned`  is `false` by default if not set

Reply:
Updated As Suggested.